### ISSUE TITLE
🚀 [Feature] Share via Social Media

### ISSUE THREAD

**Issue by ulfgebhardt** [dreammall-earth/dreammall.earth#2137]
## 🚀 Feature

Share via Social Media

Claudia wished for an option to share the Dreammall via Social Media.

It is to clarify what is to share and where before we can implement this

**Comment by ulfgebhardt** [comment]
It was decided with @Hardy-Hart-am-Start to postpone this, hence we close this in favor of https://github.com/dreammall-earth/dreammall.earth/issues/2220
